ray-on-gke/platform/main.tf

```diff
@@ -44,6 +44,7 @@ provider "kubectl" {
   cluster_ca_certificate = base64decode(
     data.google_container_cluster.ml_cluster.master_auth[0].cluster_ca_certificate
   )
+  load_config_file = false
```

Comment:
Doesn't this break when running the template locally?

Reply:
Tested locally. The first run will pass. But it will break local run when we modify and run terraform apply again. 

However it will break the cloudbuild if we dont add this. https://pantheon.corp.google.com/cloud-build/builds;region=us-central1/dd9b4a21-3497-4178-812d-bfc097e49446?e=-13802955&mods=logs_tg_prod&project=gke-ai-eco-dev
```
Error: kube-system/nvidia-driver-installer failed to create kubernetes rest client for update of resource: Get "http://localhost/api?timeout=32s": dial tcp 127.0.0.1:80: connect: connection refused
```
I did some searches and got the solution that setting `load_config_file` would be good.
maybe we can make `load_config_file` this a variable and set to false only when running cloudbuild?
cc @richardsliu

Reply:
ok was able to fix this issue by setting load_config_file from ENV variable in cloudbuild.